Sprout/Views/PostCell.m

```diff
@@ -20,12 +20,18 @@ - (void)setSelected:(BOOL)selected animated:(BOOL)animated {
 }
 
 - (void) refreshData {
+    PFUser *user = self.post.author;
+    
     [self.profileImage setImage:nil];
     
-    PFUser *user = self.post.author;
     self.usernameLabel.text = user.username;
     self.captionLabel.text = self.post.caption;
     self.createdAtLabel.text = self.post.createdAt.shortTimeAgoSinceNow;
+    self.progressLabel.text = [NSString stringWithFormat:@"I completed %@ of %@ tasks today!", self.post.completedTasks, self.post.totalTasks];
+    PFFileObject *photo = user[@"profileImage"];
+    [photo getDataInBackgroundWithBlock:^(NSData * _Nullable imageData, NSError * _Nullable error) {
```

Comment:
This is interesting. I'd bet there's a bug here if you have a ton of cells and scroll fast. Do you see what it is?


Timeline:
| cell shown, fetch 1 starts
|
| scroll, scroll,
|
| cell 1000 shown, fetch 1000 starts
| fetch 1 completes <---- what happens here?